strings/traumas.json

```diff
@@ -1,153 +1,111 @@
-        "DON'T EVER TUCH ME",
-        "@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS",
-        "its gonna be like 9/11 all over again but insitaead is gonna be a bitc",
-        "@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????"
+        "@pick(semicolon) Why is the floor spinning?",
+        "Water please...",
+        "Do you have brain damage?",
+        "Too many lights…",
+        "I feel like I can @pick(powers)!",
+        "Welcome to the station crew, enjoy your stay.",
+        "Welcome to the crew station, stay your enjoy.",
+        "I wanna pet a monkey...",
+        "I wanna go home...",
+        "I wanna be a cowboy baby!",
+        "Set sail!",
+        "@pick(semicolon) Where's my ID!",
+        "Where's my tail!",
+        "If I had a tail... @pick(qotsareference)",
+        "I'm late! I'm late! For a very important date!",
+        "I think I brained my damage...",
+        "Take it easy!",
+        "Ow!!",
+        "Too many faces...",
+        "I want... a @pick(muffin) muffin...",
+        "Poppy |seed!!|",
+        "I wanna dig a hole...",
+        "Space is so cool...",
+        "Rats...",
+        "Why is the room shaking?",
+        "Welcome to @pick(barname)...!",
+        "Call the shuttle!",
+        "Medical is scary...",
+        "I don't know... That's scary...",
+        "You're a big guy...",
+        "Cheese...",
+        "I need thoughts and prayers...",
+        "Get away from me!",
+        "The air tastes all itchy...",
+        "@pick(semicolon) Help me!",
+        "What does that mean...",
+        "Who the hell do you think I am...?",
+        "Oh hi, @pick(name)!",
+        "@pick(semicolon) Ai... I want a @pick(muffin) muffin...",
+        "This is my dorm! It was made for me!",
+        "Who the hell do you think you are?",
+        "Get out...",
+        "Moth party!",
+        "I love moths...",
+        "Ugh... My head hurty...",
+        "Pew pew! Pshoom!",
+        "For you...",
+        "Pow pow! Dakka dakka dakka!",
+        "Braaains...",
+        "I wonder how you make crème brûlée...",
+        "No... No needles!",
+        "@pick(semicolon)Tajaran has wares... if you have coin...",
+        "@pick(semicolon)I've got balls of steel!",
+        "I can take a whole deathsquad! ...In a fight!",
+        "Happy birthday to you...~",
+        "@pick(semicolon) This reminds me of 2484...",
+        "Cubic symmetry four day station...",
+        "Hehe... Bananas...",
+        "@pick(semicolon) N-NYAAAAAA~",
+        "Don't ever touch me!",
+        "I need credits... About three fifty...",
+        "The singularity is coming!!"
     ],
 
-    "mutations": [
-        "telikesis",
-        "halk",
-        "eppilapse",
-        "kamelien",
-        "eksrey",
-        "glowey skin",
-        "fungal tb",
-        "stun gloves"
+    "powers": [
+        "fly",
+        "do anything",
+        "breathe fire",
+        "disappear",
```

Comment:
dont see why any mutations needed replacement, they're just slurred and mis-pronounced mutation names